parsing/pprintast.ml

```diff
@@ -1261,13 +1261,22 @@ and binding ctxt f {pvb_pat=p; pvb_expr=x; pvb_constraint = ct; _} =
       | _ -> pp f "=@;%a" (expression ctxt) x
   in
   match ct with
-  | Some { locally_abstract_univars=[]; typ } ->
+  | Some (Constraint { locally_abstract_univars = []; typ }) ->
       pp f "%a@;:@;%a@;=@;%a"
         (simple_pattern ctxt) p (core_type ctxt) typ (expression ctxt) x
-  | Some { locally_abstract_univars; typ } ->
+  | Some (Coercion {ground=None; coercion }) ->
+      pp f "%a@;:>@;%a@;=@;%a"
+        (simple_pattern ctxt) p (core_type ctxt) coercion (expression ctxt) x
+  | Some (Coercion {ground=Some ground; coercion }) ->
+      pp f "%a@;:%a@;:>@;%a@;=@;%a"
+        (simple_pattern ctxt) p
+        (core_type ctxt) ground
+        (core_type ctxt) coercion
+        (expression ctxt) x
+  | Some (Constraint { locally_abstract_univars = vars; typ }) ->
```

Comment:
Nitpick: Could the `Constraint` cases be put next to each other? That makes for easier reading and updating.

Reply:
Good point, thanks!